ls1intum/Artemis
# added attachment download access token

### Checklist

### Motivation and Context
Currently the attachments are downloaded with a static link but without any access control. To prevent misuse of the attachments file an access token should be available.

### Description
To download an attachment file an access token is now required. It needs to be generated before with an extra API call and is only valid for a short time period. 

### Steps for Testing

1. Log in to ArTEMiS
2. Try to download an attachment
3. The behaviour should not be changed

### Screenshots
